spec/services/nytimes_service_spec.rb

```diff
@@ -0,0 +1,31 @@
+require "rails_helper"
+
+RSpec.describe NytimesService do
+
+	describe "articles" do
+		it "saves articles in db" do
+			VCR.use_cassette("articles") do
+        NytimesService.new.articles
+			end
+
+			expect(Article.first.title).to(
+				eq("California Imposes Water Restrictions, in First for State")
```

Comment:
Indent the first parameter one step more than the previous line.<br>Tab detected.